config.json

```diff
@@ -40,7 +40,7 @@
       "description": "MsBuild logging options.",
       "valueType": "passThrough",
       "values": [],
-      "defaultValue": "/l:BinClashLogger,Tools/net46/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log"
```

Comment:
Yes, it did. It is getting failed during compilation wrappers. I should delete this because option `-MsBuildEventLogging=\" \"` disables 'binclash logger' is ignored in Linux and working on Windows. Anyways on windows this logger is turned off so it has no impact.